packages/sync/tsconfig.build.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig/isomorphic.json",
```

It would be nice if this package could be isomorphic, but I don't think its deps really permit it.
```suggestion
  "extends": "../../tsconfig/node.json",
```

---

good catch